### ISSUE TITLE
No properties available

### ISSUE THREAD

**Issue by super-ibby** [tryptech/obs-zoom-and-follow#70]
Hi, first of all, thank you for this plugin. I've heard good things.

Getting to the point, I'm having difficulty getting this to work. At the point of adding the script to OBS, the window displays a message saying **No properties available**. I've provided screenshots, below. Also, clicking **Script Log** shows an empty log window.

### My environment

- MacOS 12.6 (x64).
- OBS version 28.0.2.
- `pywinctl` version 0.0.38.
- The version of your script as of today (commit [1d17b5d](https://github.com/tryptech/obs-zoom-and-follow/commit/1d17b5d688e8b3cdf4c60efe06f4f4a7c7b5d47b)).

### Screenshots

The issue

<img width="772" alt="Screenshot 2022-09-25 at 9 54 59 PM" src="https://user-images.githubusercontent.com/112219012/192180380-76429fd0-6351-4fd8-8742-b436f72fbfb2.png">

Python path

<img width="747" alt="Screenshot 2022-09-25 at 9 54 07 PM" src="https://user-images.githubusercontent.com/112219012/192180490-52e05693-caf9-47c6-9c52-1be1d8cff0ed.png">

`pywinctl` version

<img width="747" alt="Screenshot 2022-09-25 at 9 54 22 PM" src="https://user-images.githubusercontent.com/112219012/192180547-305914dc-a24c-4d11-8801-dd32f0f3c16f.png">

**Comment by tryptech** [comment]
Hmm... from my experience with obs python, when no properties show up, it's either the python is broken in some way, or the python isn't loaded in properly.

Forgive me as I don't have a way to test on MacOS, but shouldn't the python install path be /Users/ibby/miniconda3/bin/python based on what I see in the terminal screenshot?

**Comment by super-ibby** [comment]
Thanks.

When prompted for the Python Path location, it only lets me select a directory. The "python" in the path I selected is the interpreter, not a directory.

After some further digging, I found the OBS log file (noob here). It looks like it is in fact complaining about python. [I've created an issue on the OBS repo](https://github.com/obsproject/obs-studio/issues/7473), so I'll close this here.

Thanks again,
Ibby
